src/styles/_catalog.scss

```
@@ -13,10 +14,23 @@
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
   }
 
-  &__card:not(:nth-child(4n)) {
-    margin-right: 48px;
+  @media (max-width: 1023px) {
+    & {
+      grid-template-columns: auto auto auto;
+    }
```

```suggestion
    grid-template-columns: auto auto auto;
```
this nesting is redundant here - FYI (no need to push changes or re-request review, just take it into account in the future)